.mergify.yml

```diff
@@ -0,0 +1,20 @@
+pull_request_rules:
+  - name: automatic merge when CI passes and 2 reviews approved
+    conditions:
+      - "#approved-reviews-by>=2"
+      - status-success=continuous-integration/travis-ci/pr
+      - status-success=continuous-integration/travis-ci/push
+      - base=master
+    actions:
+      merge:
+        method: merge
+  - name: delete head branch after merge
+    conditions:
+      - merged
+    actions:
+      delete_head_branch: {}
```

Comment:
This part has Github itself in the Settings and it is turned on already. :)

Reply:
Oh, apologies. I thought it was being done manually. I will delete this rule. Should I keep the one where head branch is deleted when it is `closed`?

Reply:
✅